.github/workflows/deploy.yml

```diff
@@ -17,9 +16,7 @@ jobs:
       postgres_unit_testing: true
       db_name: fsd_app_store_test
       deploy_to_dev: false
-      users: 2
-      spawn-rate: 2
-      run-time: 11s
```

Comment:
Can you re-add users, run-time and spawn-rate please? @gio-karunakaran

Some repos needs to have different locust configs for the performance tests. This will override what we have set in the workflows repo for this repo only.

Reply:
as discussed with @rayhan15243, a separate ticket will be raised to set unique values for spawn rates, users and run time 👍🏾

Reply:
Thanks @gio-karunakaran

Reply:
https://digital.dclg.gov.uk/jira/browse/FS-898